src/test/java/com/iexec/worker/compute/post/web2/UploaderServiceTests.java

```diff
@@ -166,15 +160,9 @@ void shouldNotUploadToIpfsWithIexecProxySincePostFailed() throws IOException {
         final String baseUrl = "baseUrl";
         final String fileToUpload = Files.createFile(Path.of(tmpFolder.getAbsolutePath(), "fileToUpload.zip")).toString();
 
-        final RestTemplate restTemplate = mock(RestTemplate.class);
-        final ResponseEntity<String> response = ResponseEntity.notFound().build();
-        when(restTemplate.postForEntity(eq(baseUrl), any(), eq(String.class))).thenReturn(response);
-
-        when(uploaderService.createRestTemplate()).thenReturn(restTemplate);
-
         final PostComputeException exception = assertThrows(PostComputeException.class, () -> uploaderService.uploadToIpfsWithIexecProxy(COMPUTED_FILE, baseUrl, IPFS_TOKEN, fileToUpload));
         assertEquals(POST_COMPUTE_IPFS_UPLOAD_FAILED, exception.getStatusCause());
-        assertEquals(String.format("Can't uploadToIpfsWithIexecProxy (result proxy issue)[taskId:%s, status:%s]", TASK_ID, response.getStatusCode()), exception.getMessage());
+        assertEquals(String.format("Can't uploadToIpfsWithIexecProxy (result proxy issue)[taskId:%s]", TASK_ID), exception.getMessage());
     }
     //endregion
-}
\ No newline at end of file
+}
```

Comment:
I think it's a best practice to keep a blank line at the end of a code file 
```suggestion
}

```

Reply:
This is even a convention for almost all files stored in Git which follows Unix conventions here.